rosdep/python.yaml

```diff
@@ -6965,6 +6965,13 @@ python3-retrying:
     pip:
       packages: [retrying]
   ubuntu: [python3-retrying]
+python3-rocker:
+  alpine: [rocker]
```

Comment:
I don't see this on alpine either: https://pkgs.alpinelinux.org/packages?name=rocker&branch=edge